service/src/main/java/uk/gov/hmcts/reform/fpl/utils/ElementUtils.java

```diff
@@ -132,4 +132,10 @@ public static UUID getDynamicListSelectedValue(Object dynamicList, ObjectMapper
             .orElse(null);
     }
 
+    public static DynamicListElement getDynamicListSelectedElement(Object dynamicList, ObjectMapper mapper) {
```

Comment:
I don't think this will work, the issue was is that xui/ccd is only sending us the code portion of the selected item, so if this is the case then it cannot serialise the dynamic list using the object mapper as it would not be able to map to the correct values.

Reply:
I've added a new method in ElementUtils to return the selected element code in string format